projects/cdk/pipes/is-present/is-present.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import {Pipe, PipeTransform} from '@angular/core';
+
+import {tuiIsPresent} from '../../utils';
+
+@Pipe({name: `tuiIsPresent`})
+export class TuiIsPresentPipe implements PipeTransform {
+    transform<T>(value: T | null): value is T {
```

Comment:
`transform<T>(value?: T | null): value is T {`

Reply:
yeah, my bad

Reply:
<img width="1303" alt="image" src="https://user-images.githubusercontent.com/12021443/198561133-a0a49a20-b45c-497d-8cb1-6b3270556476.png">